client/src/shared/components/SideBar/SideBar.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import Link from 'next/link';
+
```

Comment:
- since this is somewhat a section or part of a page. 
- can you just move this to `layouts` folder.
- remove it in the components demo

Reply:
I already moved this component to layouts sir

Reply:
it's still under `components` though 
![image](https://user-images.githubusercontent.com/73863685/223075461-5b7d2cf1-2f2a-4969-b24a-c68728291aa3.png)

Reply:
ah my bad, its completely different file, Ill make the changes now
![image](https://user-images.githubusercontent.com/115448429/223077237-e9a3989d-9c98-4ee1-99ea-6ed0ee1abbf0.png)